tensorflow/c/c_api.h

```diff
@@ -1662,6 +1662,29 @@ TF_CAPI_EXPORT extern TF_Buffer* TF_GetAllRegisteredKernels(TF_Status* status);
 TF_CAPI_EXPORT extern TF_Buffer* TF_GetRegisteredKernelsForOp(
     const char* name, TF_Status* status);
 
+// --------------------------------------------------------------------------
+// Server functionality.
+
+// Server.
+typedef struct TF_Server TF_Server;
+
+// Creates new server.
+TF_CAPI_EXPORT extern TF_Server* TF_NewServer(const void* proto,
```

Comment:
Here I'd document what "proto" and "proto_len" is (a serialized `ServerDef` proto apparently?).
And also talk about how users should use the returned object.